src/layouts/Base.astro

```diff
@@ -29,11 +34,26 @@ const jsonLDSchema = JSON.stringify(
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/ico" href="/favicon.ico" />
 		<meta name="generator" content={Astro.generator} />
+		<meta
+			name="description"
+			content="Gain practical experience by working remotely on real world projects with other early-career developers."
+		/>
+		<meta property="og:title" content="The Collab Lab" />
+		<meta
+			property="og:description"
+			content="Gain practical experience by working remotely on real world projects with other early-career developers."
+		/>
+		<meta
+			property="og:image"
+			content={`${canonicalURL.href}img/participate-developers-collage.webp`}
```

Comment:
This [article from April](https://www.ctrl.blog/entry/webp-ogp.html) says WebP isn't officially supported in OG tags. Do we know it's safe to do so?

Reply:
Interesting! The article also claims that while Facebook says not to use webp, it still works. I'll switch to png to be sure.